now.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "jesses.io",
+  "version": 2
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)